components/feature/session/src/test/java/mozilla/components/feature/session/SettingsUseCasesTest.kt

```diff
@@ -41,16 +41,16 @@ class SettingsUseCasesTest {
 
     @Test
     fun `UpdateSettingUseCase will update all sessions`() {
-        val autoplaySetting = object : UpdateSettingUseCase<Boolean>(settings, sessionManager) {
+        val allowFileAccessSetting = object : UpdateSettingUseCase<Boolean>(settings, sessionManager) {
             override fun update(settings: Settings, value: Boolean) {
-                settings.allowAutoplayMedia = value
+                settings.allowFileAccess = value
             }
         }
 
-        autoplaySetting(true)
-        verify(settings).allowAutoplayMedia = true
-        verify(engineSessionA.settings).allowAutoplayMedia = true
-        verify(engineSessionB.settings).allowAutoplayMedia = true
+        allowFileAccessSetting(true)
```

Comment:
I'm not quite aware of the context of this change, could you help me to clarify?

Reply:
It's possible that I'm misunderstanding, but based on the name of the test (`UpdateSettingUseCase will update all sessions`) it appeared that this test was setting an arbitrary property and ensuring that it was updated on all sessions.  Instead of removing the test, it seemed more appropriate to switch it over to a new property.

Reply:
Got it, what confused me was that most of the test was related to autoplay 👍